.samples/prism-mock-service.yaml

```diff
@@ -0,0 +1,11 @@
+displayName: Prism Mock Service
+description: Simplified petstore API based mock service creation artifacts.
```

Comment:
What if we say "Simplified petstore API based mock service"?

Reply:
If we're explaining the service we can create from this directory, +1.

If we're explaining the directory itself, it would be the `artifacts`.

Reply:
Choreo user will be interested in the service IMO. Hence better to go with the service focused description.